exonum-java-binding-common/src/main/java/com/exonum/binding/common/message/MessageType.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2018 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.exonum.binding.common.message;
+
+/**
+ * Message types used in Exonum framework.
```

Comment:
I’d probably keep this although it is not needed at the moment in case we will ever need to access some of the system messages, but make it package private for now.